rskj-core/src/main/java/co/rsk/core/bc/BlockChainImpl.java

```diff
@@ -184,6 +184,14 @@ public long getSize() {
      */
     @Override
     public ImportResult tryToConnect(Block block) {
+        if (block == null)
+            return ImportResult.INVALID_BLOCK;
+
+        if (!block.isSealed()) {
+            panicProcessor.panic("unsealedblock", String.format("Unsealed block %s %s", block.getNumber(), Hex.toHexString(block.getHash())));
+            block.seal();
```

Comment:
Ok, let's remember to delete this check once the inmutable block change is applied